util/jsonschema.py

```diff
@@ -0,0 +1,42 @@
+from typing import Any, Dict, List
```

Comment:
I have a small fear that over time it is going to become difficult to design an interface that allows us to use all the features of json schema that we need. A small fear though...

Reply:
We can always mix and match.